playbooks/common/openshift-cluster/upgrades/v3_8/upgrade.yml

```diff
@@ -54,5 +54,6 @@
       state: started
 
 - import_playbook: ../upgrade_nodes.yml
```

Comment:
Why are we using this boolean here?

Reply:
Added a new TODO, as I assumed only OCP 3.8 should be restricted, but not Origin

Reply:
Scratch that, neither Origin nor OCP 3.8 are supported